coalib/settings/Setting.py

```diff
@@ -13,6 +14,22 @@ def path_list(obj, *args, **kwargs):
     return obj.__path_list__(*args, **kwargs)
 
 
+def glob(obj, *args, **kwargs):
+    """
+    Creates a path in which all special glob characters in all the
+    parent directories in the given setting are properly escaped.
+    """
```

Comment:
Btw please add param documentation for `obj, *args, **kwargs`, there's the question what objects can be used with `glob()`, and what `*args` and `**kwargs` can be used for.